tests/provision/bootc/test.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+IMAGE_NEEDS_DEPS="localhost/tmt-bootc-needs-deps"
+IMAGE_NEEDS_DEPS_PLAN="$(pwd)/data/image_needs_deps.fmf"
+IMAGE_INCLUDES_DEPS="localhost/tmt-bootc-includes-deps"
+IMAGE_INCLUDES_DEPS_PLAN="$(pwd)/data/image_includes_deps.fmf"
+
+CONTAINERFILE_NEEDS_DEPS="$(pwd)/data/needs_deps.containerfile"
+CONTAINERFILE_NEEDS_DEPS_PLAN="$(pwd)/data/containerfile_needs_deps.fmf"
+CONTAINERFILE_INCLUDES_DEPS="$(pwd)/data/includes_deps.containerfile"
+CONTAINERFILE_INCLUDES_DEPS_PLAN="$(pwd)/data/containerfile_includes_deps.fmf"
+
+
+rlJournalStart
+    rlPhaseStartSetup
+        # cleanup previous runs
+        test -d /var/tmp/tmt/testcloud && rlRun "rm -rf /var/tmp/tmt/testcloud"
```

Comment:
If we need to do a complete cleanup, what about using `TMT_WORKDIR_ROOT` instead to choose a different workdir root? We're trying to write tests so that they can be easily/safely executed on user laptop as well. This would remove for example all fetched images.